source/utilities.tex

```diff
@@ -2664,6 +2910,82 @@
 result of the comparison.
 \end{note}
 
+\rSec2[tuple.common.ref]{\tcode{common_reference} specialization}
```

Comment:
```suggestion
\rSec2[tuple.common.ref]{\tcode{common_reference}-related specializations}
```

Reply:
Added, but "`common_reference`-related" isn't a term, so didn't include the "-".

Reply:
Good point. I meant "Specializations related to `common_reference`", but the extensive use of hyphens in standard terms certainly makes it confusing.